crypto/x509/x509_err.c

```diff
@@ -20,7 +20,7 @@ static const ERR_STRING_DATA X509_str_reasons[] = {
     {ERR_PACK(ERR_LIB_X509, 0, X509_R_BAD_X509_FILETYPE), "bad x509 filetype"},
     {ERR_PACK(ERR_LIB_X509, 0, X509_R_BASE64_DECODE_ERROR),
     "base64 decode error"},
-    {ERR_PACK(ERR_LIB_X509, 0, X509_R_CANT_CHECK_DH_KEY), "cant check dh key"},
+    {ERR_PACK(ERR_LIB_X509, 0, X509_R_CANT_CHECK_DH_KEY), "can't check dh key"},
```

Comment:
Don't change this. It's auto-generated

Reply:
... and reverted too.

Reply:
This needs to be changed at both places. Then the autogeneration will not overwrite it.

Reply:
They are autogenerated and written to openssl.txt in the first pass but once the entry is placed in openssl.txt it is kept as there. If you change just the .c file, it will get overwritten by the text from openssl.txt.

Reply:
So if you change an error message in a C file, `crypto/err/openssl.txt` will be overwritten, and if you change `crypto/err/openssl.txt` the C file will be overwritten. Duplication of information doesn't make much sense in, general, and this `crypto/err/openssl.txt` file makes even less sense. What is the purpose of such a bizarre mechanism?

Reply:
No, you misunderstand. The openssl.txt file is the primary source. The .c files are autogenerated, but they are still in the source repo because the tool that generates them works only on unix-like platforms.

Reply:
The openssl.txt is updated and the message autogenerated by the tool only if the reason code is not there yet.

Reply:
I see, makes sense. Thank you for the explanation.

If I understand you correctly, I must update not only `crypto/err/openssl.txt`, the primary source,  but also relevant C files so that they are correctly updated when building on non-unix platforms.